visual_behavior/visualization/qc/run_save_all_container_plots.py

```diff
@@ -20,7 +21,9 @@
                 }
 
 
-container_ids = data_loading.get_ophys_container_ids(release_data_only=True)
+container_ids = loading.get_ophys_container_ids(include_failed_data=False, release_data_only=False,
```

Comment:
this is a general question, I have noticed this throughout the code, but should release_data_only be set to True? What is the reasoning behind defaulting to False?

Reply:
At this point it probably makes sense to set it to True by default